CONTRIBUTING.md

```diff
@@ -62,3 +63,47 @@ variable:
 ```bash
 $ export EXERCISES_API_URL=http://localhost:9292
 ```
+
+### Track Anatomy
+
+Each track should have the following structure:
+
+```bash
+├── .gitignore
+├── .travis.yml
+├── LICENSE
+├── README.md
+├── SETUP.md
+├── bin
+│   └── fetch-configlet
+├── config.json
+├── docs
+│   ├── ABOUT.md
+│   ├── INSTALLATION.md
+│   ├── LEARNING.md
+│   ├── RESOURCES.md
+│   └── TESTS.md
+└── exercises
+    └── hello_world
+        ├── hello_world_example.file
+        ├── hello_world.file
+        └── hello_world_test.file
+```
+
+* `LICENSE` - The MIT License (MIT)
+* `README.md` - a thorough explanation of how to contribute to the track.
+* `SETUP.md` - ???
```

Comment:
When you fetch an exercism exercise, it includes a README which is put together from a bunch of different pieces, including the stuff in SETUP.md -- it's language-specific, problem-agnostic stuff (how to run the test suite, where to find help).